src/citation/views/citation_entry_view.py

```diff
@@ -23,6 +26,40 @@ def list(self, request):
 
     def retrieve(self, request):
         pass
+    
+    @action(detail=False, methods=["post"], permission_classes=[IsAuthenticated])
```

Comment:
We need proper permission handling, otherwise users can upload to any org/project.